yt_dlp/networking/_helper.py

```diff
@@ -119,6 +120,7 @@ def make_ssl_context(
     use_certifi=True,
 ):
     context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+    context.keylog_filename = os.environ.get('SSLKEYLOGFILE')
```

Comment:
I think a pattern like this would be better:
```suggestion
    if keylog_filename := os.getenv('SSLKEYLOGFILE'):
        context.keylog_filename = keylog_filename
```

Reply:
We should make it identical to CPython imo

https://github.com/python/cpython/blob/8d42c57789b11f838da40cdc3f129b2115368dd9/Lib/ssl.py#L714-L718

Reply:
Agree, it may not exist if on an older openssl version

Reply:
Yes that's fine, we don't need the flags and stuff. Just the check if the attribute exists

Reply:
Sorry my mistake coming from JS I thought the type was more relaxed 😉. I will review the comments and update the PR.

Reply:
> These attributes don't always exist when Python is built with older openssl versions

The `_ssl.c` module will always provide `keylog_filename` in cpython and docs have it be a required attribute. See [`PySSLContext` with `PyObject *keylog_filename`](<https://github.com/python/cpython/blob/efcc96844e7c66fcd6c23ac2d557ca141614ce9a/Modules/_ssl.c#L285-L309>)

Reply:
Why not make it identical to CPython though? Ideally, we'd call `create_default_context` and then make our changes to the returned context. The reason we can't do that is because the function gives no way to bypass certificate verification. There is no reason for us to not mirror the flags check when we easily can imo.

Reply:
We don't use the flags check elsewhere, would be inconsistent. No need to overcomplicate imo